pkg/local/vmis.go

```diff
@@ -119,6 +119,7 @@ func createInstance(binding *types.SyntheticBinding, verrazzanoURI string, enabl
 			AutoSecret:      true,
 			SecretsName:     constants.VmiSecretName,
 			CascadingDelete: true,
+			API:             vmov1.API{Replicas: 0},
```

Comment:
Can you revert this since you mentioned in the other PR that we don't need this.

Reply:
Yes, I am closing this PR